pegasus/sites.v3/code.org/public/about/annual-report.md.partial

```diff
@@ -48,13 +48,12 @@ Everyone at Code.org is grateful for your partnership in the work we've done thi
 
 <h3 class="heading-sm">See previous reports</h3>
 
-* See recent [Code.org Statistics](/statistics)
```

Comment:
Just making sure getting rid of /statistics was intentional!

Reply:
Yes we deprecated it a while ago